### PR TITLE
3014 client page title

### PR DESCRIPTION
**Pull request by barshimi**
Issue Number: Client: Page Title #3014

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

In order to reflect HTML title inside every page content, I added react helmet that collect tab title and change page title accordingly.

### CODE REVIEW

**Comment by arielweinberger** [comment]
@barshimi Please make sure you specify an issue title, a description and link it to a milestone?

File: packages/amplication-client/src/Layout/PageContent.tsx [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
 };
 
 const CLASS_NAME = "amp-page-content";
 
-function PageContent({ children, sideContent, className }: Props) {
+function PageContent({ children, sideContent, className, pageTitle }: Props) {
   return (
+    <>
+    <Helmet>
+      <title>{pageTitle || "Amplication"}</title>
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Layout/PageContent.tsx:19]:
```suggestion
      <title>Amplication{ pageTitle ? ` |  ${pageTitle}`:""  }</title>
```

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/Layout/PageContent.tsx [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
   return (
-    <div className={classNames(CLASS_NAME, className)}>
-      {sideContent && (
-        <div className={`${CLASS_NAME}__tabs`}>{sideContent}</div>
-      )}
-      <main className={`${CLASS_NAME}__main`}>{children}</main>
-    </div>
+    <>
+      <Helmet>
+        <title>{`Amplication${pageTitle ? `-${pageTitle}` : ""}`}</title>
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Layout/PageContent.tsx:19]:
```suggestion
        <title>{`Amplication${pageTitle ? ` | ${pageTitle}` : ""}`}</title>
```

Comment (yuval-hazaz) [reply packages/amplication-client/src/Layout/PageContent.tsx:19]: instead of dash, I prefer using a pipe with spaces before and after

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**